pwa/src/hooks/mapping.ts

```diff
@@ -53,5 +53,21 @@ export const useMapping = (queryClient: QueryClient) => {
       },
     });
 
-  return { getAll, getOne, remove, createOrEdit };
+  const testSimpleMapping = (mappingId: string) =>
+    useMutation<any, Error, any>(API.Mapping.testSimpleMapping, {
+      onSuccess: async () => {
+        toast.success("Request succesfully tested");
```

Comment:
We already have a toast in the mapping resource (with Mapping succesfully tested), do we need this one as well?

I see that Mappings, Sources and Synchronization hooks are the only hooks where we send toasts, otherwise they're all being set via the apiService.

Might be a good idea to remove these toasts here, and let the apiServices handle them, to keep the consistency.